pkg/tenantctl/controller.go

```diff
@@ -184,21 +184,21 @@ func (c *Controller) syncHandler(key string) error {
 
 	tenant := origTenant.DeepCopy()
 
-	c.logPrintln(tenant, "started reconciliation")
-	defer c.logPrintln(&api.Tenant{ObjectMeta: *tenant.ObjectMeta.DeepCopy()}, "finished reconciliation")
+	klog.V(4).Infof(c.formatLog(tenant, "started reconciliation"))
+	defer klog.V(4).Infof(c.formatLog(&api.Tenant{ObjectMeta: *tenant.ObjectMeta.DeepCopy()}, "finished reconciliation"))
```

Comment:
```suggestion
	if klog.V(4).enabled() {
		defer klog.V(4).Infof(c.formatLog(&api.Tenant{ObjectMeta: *tenant.ObjectMeta.DeepCopy()}, "finished reconciliation"))
	}
```

Reply:
You mentioned it before. We think it is better readable as it is and the formatLog overhead does not justify the if block.
But I'm also fine with the if block 🙂